tidalapi/page.py

```diff
@@ -305,37 +359,34 @@ def __init__(self, session, json_obj):
         self.text = json_obj["text"]
         self.featured = bool(json_obj["featured"])
 
-    def get(self):
+    def get(self) -> Union["Artist", "Playlist", "Track", "UserPlaylist", "Video"]:
         """Retrieve the PageItem with the artifact_id matching the type.
 
         :return: The fully parsed item, e.g. :class:`.Playlist`, :class:`.Video`, :class:`.Track`
         """
         if self.type == "PLAYLIST":
-            result = self.session.playlist(self.artifact_id)
+            return self.session.playlist(self.artifact_id)
         elif self.type == "VIDEO":
-            result = self.session.video(self.artifact_id)
+            return self.session.video(self.artifact_id)
         elif self.type == "TRACK":
-            result = self.session.track(self.artifact_id)
+            return self.session.track(self.artifact_id)
         elif self.type == "ARTIST":
-            result = self.session.artist(self.artifact_id)
-        else:
-            raise NotImplementedError("PageItem type %s not implemented" % self.type)
-
-        return result
+            return self.session.artist(self.artifact_id)
+        raise NotImplementedError(f"PageItem type {self.type} not implemented")
 
 
 class TextBlock(object):
     """A block of text, with a named icon, which seems to be left up to the
     application."""
 
-    text = ""
-    icon = ""
-    items = None
+    text: str = ""
+    icon: str = ""
+    items: Optional[List[str]] = None
 
-    def __init__(self, session):
+    def __init__(self, session: "Session"):
         self.session = session
 
-    def parse(self, json_obj):
+    def parse(self, json_obj: JsonObj) -> "TextBlock":
```

Comment:
not your code but I'm just reading this thoroughly for the first time.. This should be a staticmethod/classmethod.  I think I'll make a general refactoring issue.

Reply:
See #193

Reply:
Yeah, a lot of our typings could be simplified if we move away from the initialize-as-empty-then-hydrate-with-a-response-and-copy technique.